### PR TITLE
feat: #24 add dependabot configuration

### PR DESCRIPTION
**Pull request by vrkljam**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,10 @@
+buildscript{
```
Comment (milehimikey) [inline build.gradle:1]: Don't need this file.

File: dependencies.gradle [hunk 2]
```diff
@@ -0,0 +1,8 @@
+ext {
```
Comment (milehimikey) [inline dependencies.gradle:1]: Nor this one.

**Review by milehimikey — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+version:1
+updates:
+  -package-ecosystem: 'gradle'
+    directory: "/"
+    schedule:
+      interval:"weekly"
+    labels:
+      - "deps:"
```
Comment (tylerkron) [inline .github/dependabot.yml:8]: we'll want a space after the colon `"deps: `

**Review by tylerkron — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+version:1
```
Comment (mikeylive) [inline .github/dependabot.yml:1]: You need a space after the colon (:) here. And this should probably be `version: 2`

**Review by tkronpax8 — APPROVED** [review]

**Review by tylerkron — APPROVED** [review]

**Review by JacobBrauchler — APPROVED** [review]

**Pull request merged**